### PR TITLE
[DNS] Audit content types and SEO 

### PR DESCRIPTION
**Pull request by RebeccaTamachiro**
PCX-5668

### CODE REVIEW

File: content/dns/zone-setups/troubleshooting/dns-probe-finished-nxdomain.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: DNS_PROBE_FINISHED_NXDOMAIN
 pcx_content_type: troubleshooting
 weight: 2
 meta:
-  title: DNS_PROBE_FINISHED_NXDOMAIN - Troubleshooting zone setups
```
Comment (RebeccaTamachiro) [inline content/dns/zone-setups/troubleshooting/dns-probe-finished-nxdomain.md:6]: Screenshot 2023-03-06 at 12 07 22

File: content/dns/additional-options/analytics.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-pcx_content_type: reference
+pcx_content_type: navigation
```
Comment (marciocloudflare) [inline content/dns/additional-options/analytics.md]: Looking at the page content this seems more like a how-to. Navigation normally doesn't have this type a content, and is more like a page with a link to pages

Comment (marciocloudflare) [inline content/dns/additional-options/analytics.md]:
```suggestion
pcx_content_type: how-to
```

File: content/dns/dns-firewall/random-prefix-attacks/_index.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-pcx_content_type: navigation
+pcx_content_type: overview
```
Comment (marciocloudflare) [inline content/dns/dns-firewall/random-prefix-attacks/_index.md]:
```suggestion
pcx_content_type: concept
```

Comment (marciocloudflare) [reply content/dns/dns-firewall/random-prefix-attacks/_index.md]: Overview is reserved for landing pages. If this isn't a product inside DNS, I think it works better as a concept.

File: content/dns/manage-dns-records/reference/record-attributes.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-pcx_content_type: reference
+pcx_content_type: overview
```
Comment (marciocloudflare) [inline content/dns/manage-dns-records/reference/record-attributes.md]: If this isn't a new product inside DNS, I think it's better to change to `concept`

Comment (marciocloudflare) [inline content/dns/manage-dns-records/reference/record-attributes.md]:
```suggestion
pcx_content_type: concept
```

File: content/dns/manage-dns-records/reference/wildcard-dns-records.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-pcx_content_type: concept
+pcx_content_type: overview
```
Comment (marciocloudflare) [inline content/dns/manage-dns-records/reference/wildcard-dns-records.md]:
```suggestion
pcx_content_type: concept
```

Comment (marciocloudflare) [reply content/dns/manage-dns-records/reference/wildcard-dns-records.md]: Not a landing page. I think it's better as a concept, as originally.

**Review by marciocloudflare — CHANGES_REQUESTED** [review]

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**